specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2024-07-01/amlfilesystem.json

```diff
@@ -0,0 +1,2266 @@
+        "operationId": "autoExportJobs_Delete"
+      },
+      "get": {
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AmlFilesystemNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/AutoExportJobNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns the auto export job object corresponding to amlFilesystemName.",
+            "schema": {
+              "$ref": "#/definitions/AutoExportJob"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "autoExportJobs_Get": {
+            "$ref": "./examples/autoExportJobs_Get.json"
+          }
+        },
+        "tags": [
+          "exportJobs"
+        ],
+        "description": "Returns an auto export job.",
+        "operationId": "autoExportJobs_Get"
+      },
+      "put": {
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/AmlFilesystemNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AutoExportJobNameParameter"
+          },
+          {
+            "description": "Object containing the user-selectable properties of the auto export job. If read-only properties are included, they must match the existing values of those properties.",
+            "in": "body",
+            "name": "autoExportJob",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AutoExportJob"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The auto export job has been created or updated.",
```

Comment:
>The auto export job has been created or updated. [](http://example.com/codeflow?start=28&length=48)

200 is for update (resource already exists) and 201 is for create (brand new resource). They should both be async (or sync).